src/panoramic/cli/local/writer.py

```diff
@@ -125,9 +126,9 @@ def delete_field(self, field: PanoField):
         """Delete field from local filesystem."""
         assert field.file_name is not None
 
-        if field.data_source is not None:
+        if field.package is not None:
```

Comment:
I think we should be using `package` here instead of data_source right?

Reply:
thought we already fixed that, good spot